doc/src/guide/connection_draining.asciidoc

```diff
@@ -67,3 +67,33 @@ after DrainTimeout ->
 end,
 ok = ranch:stop_listener(Ref).
 ----
+
+If you want to drain listeners automatically as part of your application shutdown routine,
+just put it in the `prep_stop/1` function of your application module and export it.
+
+.Draining listeners automatically on application shutdown
+
+[source,erlang]
+----
+-module(my_app).
+
+-behavior(application).
+
+-export([start/2]).
+-export([prep_stop/1]).
+-export([stop/1]).
+
+start(_StartType, _StartArgs) ->
+	{ok, _} = application:ensure_all_started(ranch),
```

Comment:
Remove this line it's supposed to be in the dependencies.

Reply:
Done.